src/Logging/Logger.php

```diff
@@ -112,6 +131,27 @@ public static function emergency(string $sMessage, LogContext $logContext, array
 
     private static function logContext(string $sLevel, string $sMessage, LogContext $logContext, array $aMetadata = []): void
     {
-        self::log($sLevel, $sMessage, \array_merge([self::CONTEXT => (string)$logContext], $aMetadata));
+        try {
+            $aDebugBacktrace = \debug_backtrace();
```

Comment:
is there a sense:
`$aDebugBacktrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);`

Reply:
in fact we do not need `args` so it make sense :+1:

Reply:
it removes not only `args` but also `object`, which is required, from the trace so we can't use DEBUG_BACKTRACE_IGNORE_ARGS

Reply:
1. If we really need to save memory, function `debug_backtrace()` expects bitmask as the parameter. So, this line will do the trick. If we don't, let's close this PR.
```php
\debug_backtrace(DEBUG_BACKTRACE_PROVIDE_OBJECT|DEBUG_BACKTRACE_IGNORE_ARGS)
```